pennylane_lightning/core/src/gates/Gates.hpp

```diff
@@ -516,17 +539,14 @@ static auto getSingleExcitation(T angle) -> std::vector<ComplexT<T>> {
             ZERO<ComplexT, T>(),
             ZERO<ComplexT, T>(),
             ZERO<ComplexT, T>(),
```

Comment:
I actually added these blank lines while developing these gates for a better representation of the underlying matrix. If you are happy with them, let's revert them back. Otherwise, feel free to ignore my suggestion ;)   
```suggestion
            ZERO<ComplexT, T>(),

```

Reply:
The latest merge of master was messy, so it is possible some changes were reverted. Let me change it again.